internal/rpmmd/repository.go

```diff
@@ -150,6 +156,39 @@ func LoadRepositories(confPaths []string, distro string) (map[string][]RepoConfi
 		return nil, err
 	}
 
+	if strings.HasPrefix(distro, "rhel-8") {
+		var sslClientKey string
+		var sslClientCert string
+
+		regexKey := regexp.MustCompile("^[0-9]*-key.pem")
+		regexCert := regexp.MustCompile("^[0-9]*.pem")
+
+		entitlementFiles, err := ioutil.ReadDir("/etc/pki/entitlement/")
+		if err != nil {
+			return nil, &RepositoryError{"LoadRepositories failed: unable to read /etc/pki/entitlement/"}
```

Comment:
Include the error message:
```suggestion
			return nil, &RepositoryError{fmt.Sprintf("unable to read /etc/pki/entitlement/: %v", err)}
```

Reply:
I changed the error message to be this.